FreeRTOS/FreeRTOS
# Use 1500 Bytes as the MTU for all demos.

Description
-----------
Using an MTU lower than 1500 is likely to lead to erratic behavior and lost packets. Remove all cases of non-1500 byte MTUs from FreeRTOS demos.

Test Steps
-----------
Verified change in qemu mps2 demo.

Checklist:
----------


- [ X ] I have tested my changes. No regression in existing tests.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
